cloudsim-plus-testbeds/src/main/java/org/cloudsimplus/sla/ExampleMetricsWithoutNetwork.java

```diff
@@ -314,7 +316,7 @@ public ExampleMetricsWithoutNetwork() throws FileNotFoundException {
     }
 
     /**
-     * Creates the switches.
+     * Creates the DATACENTER.
      *
      * @return the switches
```

Comment:
The javadoc is stating that the method is returning switches, but it is retuning a Datacenter.